### PR TITLE
feat(ui): create CourseProgressComponent (#64)

### LINKED ISSUE #64: Create CourseProgressCard component
**Acceptance criteria:**
- Uses Chakra's `Box` component as the root element.
- Its props extend `BoxProps` from Chakra UI.
- Has 100% width by default.
- Receives the required `stats` prop of `Statistic[]` type.
- Uses ProgressBar from Chakra UI.

**Interfaces:**
```ts
interface Statistic {
    name: string;
    value: number;
    max: number;
}
```

**Mockups:**
[Figma link](https://www.figma.com/file/ur0KLA9ZOtiodrAmpXrxh9/CodersCamp-app-and-website?node-id=645%3A1355)

### PR DESCRIPTION
**Pull request by lagdush**
Closes #64

### CODE REVIEW

File: packages/ui/src/components/CourseProgressCard/CourseProgressCard.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <Box w="100%" boxShadow="base" borderRadius="8px" padding="24px" {...props}>
+      <Flex direction="column" justify="space-between" h="100%">
+        <Typography as="p" weight="extrabold" size="xl" color="gray.900">
+          Postęp w kursie
+        </Typography>
+        {stats.map((value) => {
+          return (
+            <Flex direction="column" key={value.name}>
+              <Flex justify="space-between" pt="24px">
+                <Typography as="p" weight="medium" size="sm" color="gray.900">
```
Comment (KonradSzwarc) [inline packages/ui/src/components/CourseProgressCard/CourseProgressCard.tsx:30]: 🤔 There should be 4px of margin bottom

File: packages/ui/src/components/CourseProgressCard/CourseProgressCard.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                  {value.name}
+                </Typography>
+                <Typography as="p" size="sm" color="gray.400">{`(${value.value}/${value.max})`}</Typography>
+              </Flex>
+              <Progress value={value.value} max={value.max} />
```
Comment (KonradSzwarc) [inline packages/ui/src/components/CourseProgressCard/CourseProgressCard.tsx:35]: 🤔 There should be 2px of border radius

File: packages/ui/src/components/CourseProgressCard/CourseProgressCard.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { Flex } from '../Flex';
+import { Typography } from '../Typography';
+
+interface Statistic {
+  name: string;
+  value: number;
+  max: number;
+}
+
+type Stats = Statistic[];
```
Comment (KonradSzwarc) [inline packages/ui/src/components/CourseProgressCard/CourseProgressCard.tsx:13]: 🙃 As you use it only once maybe there is no need to create this type

File: packages/ui/src/components/CourseProgressCard/CourseProgressCard.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { Box } from '../Box';
+import { CourseProgressCard, CourseProgressCardProps } from './CourseProgressCard';
+
+const status = [
```
Comment (KonradSzwarc) [inline packages/ui/src/components/CourseProgressCard/CourseProgressCard.stories.tsx:7]: 🙃 You can name it `statsData` and re-export to tests so you wouldn't duplicate mocks

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Review by KonradSzwarc — APPROVED** [review]

**Pull request merged**